app/views/posts/index.html.erb

```diff
@@ -1 +1,21 @@
-<h1>List of Post</h1>
\ No newline at end of file
+<h1>All Posts</h1>
+<%= render 'partials/user', user: @user %>
+
+<% @user.posts.each_with_index do |post, index| %>
+  <a href="/users/<%= @user.id %>/posts/<%= post.id %>">
+    <div class="container recent_post">
+      <p class="post-num">Post #<%= index + 1 %></p>
+      <p class="post-title"><%= post.title %></p>
+      <small class="counters">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></small>
+
+      <% if post.five_recent_comments.length.positive? %>
```

Comment:
- [ ] It is not possible to see the post#index page, please fix the issues to be able to see all posts 👇🏽 

![image](https://github.com/KAYEMBAJONATHAN/Blog-app-/assets/58642949/b5c683f0-632f-4f73-804b-b9c5b125a526)